README.md

```diff
@@ -1,19 +1,112 @@
 
-## Learning Goal 
+Modules provide the same organizational structure and value for your application!
 
-This lesson introduces students to dependency management in Node.js . It should demonstrate the different ways that `require()` works — highlighting some of its quirks and virtues — and it should also introduce them to `npm`
+## Dependencies and NPM
+We'll introduce you to the concept of modules (which we'll sometimes refer to as packages) and dependencies by creating a basic app with Express - the popular Node web application framework.
 
-## Learning Objectives- Students should be able to: 
+Express can be installed into your application via NPM, Node's package (module) management system. We can use the NPM command line tool that ships with Node to install packages into our application from the NPM registry.
 
-- Pull in external dependencies for their applications
-- Spread their applications out over several files instead of writing everything in one file
-- Explain different export strategies and how to use each strategy in their application
+Let's use NPM to install our app's very first dependency: Express.
 
+Open Terminal and cd into your app's root directory.
 
-<p class='util--hide'>View <a href='https://learn.co/lessons/node-curriculum-assignment-dependency-management-readme'>Node Curriculum Assignment Dependency Management Readme</a> on Learn.co and start learning to code for free.</p>
+```
+$ cd ~/path/to/app
+```
+
+The first thing you'll want to do is create a `package.json` file which contains meta data about your app including a list of your apps dependencies. Remember: "dependencies" are just the modules that your app needs in order to function - In other words, dependencies are the modules or packages your app *depends* on.
+
+We can create a `package.json` file with ease by executing the following command in our project's root directory:
+```
+$ npm init
+```
+
+Executing this command will prompt you for a number of things, like the name and version of your application. For now, you can simply hit `return` to accept the defaults for most of the questions, with the following exception:
+```
+entry point: (index.js)
+```
+Just go ahead and enter the name of your main file then press `enter`
+
+Next we'll install the Express package:
+```
+$ npm install express --save
+```
+NPM will search the registry for the Express package and download it to the `node_modules` folder in your projects root directory. The `--save` part just tracks Express as a new dependency in your `package.json` file.
+
+Now we're ready to start building our simple Express app. Go ahead and copy this code into your `index.js`.
+
+`index.js:`
+```
+const express = require('express');
+const app = express();
+
+app.get('/', function (req, res) {
+  res.send('Hello World!');
+});
+
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000!');
+});
```

Comment:
This is a lot of code for students to parse when what we really want to focus on (and what you focus on below) is `require`. Is there a way to split this up? Maybe by working with local files first?